src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,3 +1,13 @@
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+import React from 'react';
+import './UserInfo.scss';
+import { User } from '../types/User';
+
+type Props = {
+  user: User | null,
```

Comment:
You render this component **only** when user exists
```suggestion
  user: User,
```